src/main.cpp

```diff
@@ -670,8 +675,11 @@ int main(int Argc, char *Argv[]) {
                 errorHandler.AddError("Unsupported value for --target-os, supported values are: %s",
                                       g->target_registry->getSupportedOSes().c_str());
             }
-        } else if (!strcmp(argv[i], "--disable-vectorcall")) {
-            disableVectorCall = true;
+        } else if (!strcmp(argv[i], "--no-vectorcall")) {
+            vectorCall = VectorCallStatus::disabled;
+         }
+        else if (!strcmp(argv[i], "--vectorcall")) {
+            vectorCall = VectorCallStatus::enabled;
```

Comment:
Add check to see if --vectorcall and --no-vectorcall are used together?

Reply:
I think it's ok to allow using them both and have the last one win, it's pretty standard behavior. We have it for --target as well.